src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -1743,12 +1751,9 @@ public String toString() {
 			if (isNotBlank(getMatrixCoefficients())) {
 				result.append(", Matrix Coefficients: ").append(getMatrixCoefficients());
 			}
-			if (isNotBlank(avcLevel)) {
-				result.append(", AVC Level: ").append(getAvcLevel());
+			if (isNotBlank(getVideoFormatProfile())) {
+				result.append(", Video Format Profile: ").append(getVideoFormatProfile());
```

Comment:
If it's the whole string that's make sens to me, but it's not what i saw, so i need to check it again.